src/pages/CompletedTasks.js

```diff
@@ -1,9 +1,14 @@
 import React from "react";
+import TaskList from "../components/TaskList";
+import NavBar from "../components/NavBar";
+import { MainCont } from "../styled-components/reusables";
 
 export default function CompletedTasks() {
+	const url = `http://localhost:4567/complete-tasks`;
 	return (
-		<>
-			<h1>complete</h1>
-		</>
+		<MainCont width="100%">
```

Comment:
You don't have to change it, but i see this code being repreated in your project: 

```
	<MainCont width="100%" >
			<NavBar />
			<TaskList url={url} />
		</MainCont>
``` 

This seems to me like a good material for another reusable component.